release-matrix/release-matrix.json

```diff
@@ -1,4 +1,20 @@
 [
+  {
+    "mkeReleaseVersion": "v4.1.3-alpha.1",
+    "k0rdentVersion": "1.2.0",
+    "mkeOperatorImageVersion": "v1.5.0",
+    "mkeOperatorChartVersion": "3.15.0",
+    "k0sVersion": "v1.32.6+k0s.0",
+    "isLatest": true,
+    "minimumMkectlVersion": "v4.1.3",
```

Comment:
@ry4nz - Looking at https://github.com/MirantisContainers/mke-release/pull/42/changes, shouldn't the minimumMkectlVersion be the same as release version for this PR?